BUILD_AND_INSTALL_FORKLIFT_ON_KIND.md

```diff
@@ -12,96 +12,37 @@ used as a VM migration target.
 Only the backend is installed, i.e. operator/controller/validation, without
 the UI, because it is meant to be the target for running automated tests.
 
-
-# Prerequisits
-
+# Pre-requisite
```

Comment:
`Prerequisits` is ok, no need to change it (see [our doc](https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#prerequisites))

Reply:
its Prerequisites :)